content/mqtt/index.textile

```diff
@@ -0,0 +1,103 @@
+
+On receiving this, the client is recommended to get a new token, then disconnect and reconnect with the new token themselves. If this is not done, once the token expires, the server will abruptly disconnect the connection.
+
+h3(#ssl). SSL
+
+We support both ssl and non-ssl connections (which use a different port, see above), but strongly recommend using ssl wherever possible. If you are not using ssl, note that the same restrictions apply as if you were using an Ably client without ssl. That is, connection attempts using basic authentication (i.e. an api key) are disallowed, and any namespaces which you've enabled 'require TLS' on will be inaccessible. If this is a problem for you, please do let us know at support@ably.io and we'll see what we can do.
+
+h2(#info). Extra information
+
+h3(#adapter-info). Adapter information
+
+* A protocol adapter inevitably adds some latency. Using the adapter will be a little slower than using Ably native client libraries. Typically the impact is in the low milliseconds
+* Behind the scenes, the adapter just uses the normal Ably service, so there is no problem with using MQTT and Ably client libraries side by side. You can mix and match as you like; for example, using MQTT on your IoT devices, but using the Ably Realtime API on your servers
+* While the adapter can be useful for devices which need to use MQTT, there are many benefits to using the Ably client libraries (e.g. "continuity guarantees":https://support.ably.io/solution/articles/3000044639-connection-state-recovery, "fallback host support":https://support.ably.io/solution/articles/3000044636-routing-around-network-and-dns-issues, "history":https://www.ably.io/documentation/realtime/history, "presence":https://www.ably.io/documentation/realtime/presence) require the use of the Ably client libraries. As a result, if a native Ably library is available for your platform, we recommend you consider using the Ably client libraries instead or at least make a plan to eventually transition over to Ably native client libraries
+
+h3(#mqtt-info). MQTT information
+
+* We support MQTT 3.1.1 clients only. Connection attempts using earlier protocol versions will be rejected
+* Publishing supports QoS 0 or 1. Subscribing only supports QoS 0
+* Session resumption is supported within the usual "Ably time limit of two minutes":https://support.ably.io/solution/articles/3000044639-connection-state-recovery
+* "No wildcard subscriptions of any kind are supported":https://support.ably.io/solution/articles/3000063123-can-i-attach-and-subscribe-to-channels-using-wildcards. Subscribe requests containing either single or multi-level wildcards will be nack'd
+Will messages and the Retain flag are not currently supported
```

Comment:
last bullet point missing

Also, since these four bullet points are basically all about what MQTT features we support, I'd move them up to the "Supported features" section. Someone looking to see what features the adapter supports is going to look at that section, not in "Extra information -> MQTT information"